packages/flutter_tools/bin/podhelper.rb

```diff
@@ -177,9 +177,9 @@ def flutter_install_ios_engine_pod(ios_application_path = nil)
       Pod::Spec.new do |s|
         s.name             = 'Flutter'
         s.version          = '1.0.0'
-        s.summary          = 'High-performance, high-fidelity mobile apps.'
-        s.homepage         = 'https://flutter.io'
-        s.license          = { :type => 'MIT' }
+        s.summary          = 'A UI toolkit for beautiful and fast apps.'
+        s.homepage         = 'https://flutter.dev'
+        s.license          = { :type => 'BSD' }
```

Comment:
This should be `BSD` https://github.com/CocoaPods/Specs/blob/master/Specs/c/0/4/Flutter/3.0.0/Flutter.podspec.json#L8